### PR TITLE
fix: use /go suffix in go package option

### PR DESCRIPTION
**Pull request by colin-axner**
Thanks @damiannolan for helping debug this issue. We noticed when trying to test the latest commit with ibc-go that the go option would cause the .pb.go files to be generated in the wrong package

We are unsure if there's a way to test these changes with ibc-go before merging, but we can definitely do it once there's a buf tag with this commit

### CODE REVIEW

**Comment by colin-axner** [comment]
I'm not sure why the tests are failing or if that is expected

File: proto/cosmos/ics23/v1/proofs.proto [hunk 1]
```diff
@@ -2,16 +2,17 @@ syntax = "proto3";
...
 enum HashOp {
-  // NO_HASH is the default if no data passed. Note this is an illegal argument some places.
-  NO_HASH    = 0;
-  SHA256     = 1;
-  SHA512     = 2;
-  KECCAK     = 3;
-  RIPEMD160  = 4;
-  BITCOIN    = 5;  // ripemd160(sha256(x))
+  // NO_HASH is the default if no data passed. Note this is an illegal argument
+  // some places.
```
[the remark this answers is not in the document]
Comment (tac0turtle) [reply proto/cosmos/ics23/v1/proofs.proto:9]: running make photo-all foes linting and formatting. Running make proto-gen-go is simple

**Review by tac0turtle — APPROVED** [review]

**Comment by colin-axner** [comment]
undid the formatting changes

File: proto/cosmos/ics23/v1/proofs.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (damiannolan) [reply proto/cosmos/ics23/v1/proofs.proto]: Adding the `clang-format` from ibc-go fixes `make proto-all` for me

**Review by damiannolan — APPROVED** [review]

**Review by julienrbrt — APPROVED** [review]

**Pull request merged**